manifests/config.pp

```diff
@@ -18,4 +18,13 @@
     group   => $kibana::kibana_group,
     mode    => '0660',
   }
+
+  if $kibana::plugindir {
+    file { $kibana::plugindir:
+      ensure => 'directory',
+      owner  => $kibana::kibana_user,
+      group  => $kibana::kibana_group,
+      mode   => 'o+Xr',
```

Comment:
Can we use something more "strict" (enforce permissions for owner, group and other), i.e.:

```suggestion
      mode   => '0755',
```

Then adjust the test accordingly.